Detectors/TOF/simulation/src/DigitizerTask.cxx

```diff
@@ -67,7 +69,10 @@ void DigitizerTask::Exec(Option_t* option)
   /*if (mDigitsArray)
     mDigitsArray->clear(); // this clear is now moved to the digitizer
   */
-  mDigitizer.setEventTime(mgr->GetEventTime());
+  o2::InteractionTimeRecord firstorbit(o2::InteractionRecord(0, o2::raw::HBFUtils::Instance().orbitFirstSampled), 0.0);
+  o2::InteractionTimeRecord orbit(mgr->GetEventTime());
+  orbit += firstorbit;
+  mDigitizer.setEventTime(orbit);
```

Comment:
Do you use DigitizerTask with the FairRootManager for anything? In any case, the time definition is wrong: `orbitFirstSampled` is the orbit from which the sampler starts in the o2-sim-digitizer.

Reply:
This is a an obsolete task never used
I'm going to remove it

Reply:
class removed